### ISSUE TITLE
Report a problem on this page component: some of its specifications on the Canada.ca Content & IA specification

### ISSUE THREAD

**Issue by sarasuri** [wet-boew/GCWeb#1205]
Hi,
Some of the specification outlined in the 6.3.2.3 section for the “Report a problem on this page” component on the Canada.ca Content & IA specification (v 1.3.2) appears to be specific to the Canada.ca home page and it doesn't apply to the other GCWeb pages.  For example, 
Specifications:
"When clicked, the button triggers a form which expands immediately below the button. Page
components that follow are pushed down to accommodate the form. When clicked a second
time the form will collapse. "

"When the user hits the “Send” button, the form is submitted and replaced by a thank you
message. The thank you message can be collapsed by clicking the “Report a problem with this
page” button. The form should then be reset so that users may submit any other problems they
encounter on that page."

In addition, the following specifications should be modified.
- "Placement: positioned below the Date modified component, left-aligned."
It is positioned above the Date modified component not below.

- "The submit button is labeled “Send”."
The submit button is labeled “Sumbit” not “Send”.

**Comment by DK-TBS** [comment]
Thanks @sarasuri, we're aware of this.  That button is intended to always open up a form directly on the page (as you see on the home page), but the way Canada.ca is spread across different servers means that some parts of the site weren't able to make that happen.  So until we get everything moved over to the AEM solution, there will be some parts of the site in which that button will simply link off to a separate page that has the form on it.

The differences you're seeing with the labels, and around the placement of the "date modified" component are just the result of the spec being a little ahead of reality.  The spec merely describes what we'd like to see as a final end state.  It always takes a while for the templates and components to catch up to what's been specified in the document.

**Comment by sarasuri** [comment]
Thank you for the feedback.
May be you are already aware this as well. In case, you are not.

When the user hits the “Send” button, the form is submitted and replaced by a thank you message. 
**The thank you message can be collapsed by clicking the “Report a problem with this page” button. The form should then be reset so that users may submit any other problems they encounter on that page.**

Does this mean the form is reset automatically and when user expands the form again by clicking on the button? 
Because when the form is expand the thank you message is still there. You have to refresh the page manually to get the form display on the page again. Used Firefox 50.

**Comment by DK-TBS** [comment]
Yep, we're aware of that, as well.  The spec is describing what we'd *like* to have in place, not necessarily what's in place now  :)

**Comment by zzucco** [comment]
Hi,
It seems like we can close this issue?

**Comment by jkshapiro** [comment]
See also #1208 and #1336
